### DISCUSSION TITLE
Sorry to interupt, when will the new version be released?

### DISCUSSION THREAD

**Discussion by kongyi182** [ipxe/ipxe#887] [Q&A] [answered]

**Comment by stappersg** [comment, marked as answer]
Duplicate of #679

**Comment by NiKiZe** [comment]
Last commit in git master is 3 days old https://github.com/ipxe/ipxe/commit/62a1d5c0f5bc52227ba43ccb7924694d661449a2 it is already released as git master.

**Comment by Neustradamus** [comment]
@NiKiZe: Like I have done my personal ticket, I confirm that it is needed to have new official builds!
- https://github.com/ipxe/ipxe/issues/904

Linked to:
- https://github.com/ipxe/ipxe/discussions/887
- https://github.com/ipxe/ipxe/issues/886
- https://github.com/ipxe/ipxe/pull/794
- https://github.com/ipxe/ipxe/discussions/679
- https://github.com/ipxe/ipxe/discussions/544

**Reply by NiKiZe** [reply]
You are duplicating this all over. Do you think this will change facts of how versioning is done?

To be clear, this is not acceptable behaviour.
